client/src/livingdocs-component-app/app.css

```diff
@@ -0,0 +1,192 @@
+
+.livingdocs-component-item-container {
+  width: 50%;
+  float: right;
+}
+
+.livingdocs-component-search {
+  width: calc(50% - 1px);
```

Comment:
Set `box-sizing: border-box;` for everything, then you do not have to have the calculation like this as the border is part of the box size then. See https://github.com/nzzdev/Q-editor/blob/dev/client/src/styles/layout.css#L3 and https://github.com/nzzdev/Q-editor/blob/dev/client/src/styles/layout.css#L12-L14